pages/book/functions.mdx

```diff
@@ -23,47 +29,99 @@ fun bar() {
 }
 ```
 
-## Global static functions
+## Native functions, `native` [#native]
+
+{/* TODO Top-level, mappings to FunC */}
+
+Native functions are direct bindings of FunC functions:
+
+> **Note**
+> Native functions could be also mutable and extension ones.
```

Comment:
```suggestion
> Native functions could be also mutating and extension ones.
```

Reply:
Thanks! Although I've just posted a very rough draft, and this paragraph will be annihilated and then brought back from the ashes :)